src/framework/dropdown/_dropdownOption.scss

```diff
@@ -9,4 +9,8 @@
   &.is-dropdown-option-focus {
     background-color: #f3f9fd;
   }
+
+  &.dropdownGroupFirst {
```

Comment:
BEM style would rather be `dropdownGroup--first` or even `dropdownGroup__dropdownOption--first`.